buildscripts/kokoro/xds_url_map.sh

```diff
@@ -51,17 +54,24 @@ build_test_app_docker_images() {
   cp -v "${docker_dir}/"*.Dockerfile "${build_dir}"
   cp -v "${docker_dir}/"*.properties "${build_dir}"
   cp -rv "${SRC_DIR}/${BUILD_APP_PATH}" "${build_dir}"
+  # Pick a branch name for the built image
+  if [[ -n $KOKORO_JOB_NAME ]]; then
+    branch_name=$(echo "$KOKORO_JOB_NAME" | sed -E 's|^grpc/java/([^/]+)/.*|\1|')
```

Comment:
Wrap in quotes.

Reply:
Done, and updated in two backports. PTALA.